### ISSUE TITLE
minor parsing issue

### ISSUE THREAD

**Issue by braindead-dev** [arXiv/html_feedback#5169]
### Description

says "/Description" instead of "Description"

### (Optional:) Please add any files, screenshots, or other information here.

<img width="905" height="283" alt="Image" src="https://github.com/user-attachments/assets/d92a0714-7ad3-45a3-9e00-9ec42fab7ec9" />

### (Required) What is this issue most closely related to? Select one.

Other

### Internal issue ID

ac2d8553-aec3-4499-9786-01f525c6e285

### Paper URL

https://arxiv.org/html/2501.00383

### Browser

Chrome/141.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Bookkeeping comment: This is pending the upgrades from brucemiller/LaTeXML#2666

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
